packages/core/data/oot/world_mq/dodongo_cavern_mq.yml

```diff
@@ -101,3 +119,27 @@
   locations:
     "MQ Dodongo Cavern Chest Under Grave": "true"
     "MQ Dodongo Cavern GS Near Boss": "true"
+    "MQ Dodongo Cavern Pot Before Boss 1": "true"
+    "MQ Dodongo Cavern Pot Before Boss 2": "true"
+    "MQ Dodongo Cavern Pot Before Boss Loop 1": "true"
+    "MQ Dodongo Cavern Pot Before Boss Loop 2": "true"
+    "MQ Dodongo Cavern Pot Before Boss Loop 3": "true"
+    "MQ Dodongo Cavern Pot Before Boss Loop 4": "true"
+    "MQ Dodongo Cavern Pot Before Boss Loop Side Room 1": "true"
+    "MQ Dodongo Cavern Pot Before Boss Loop Side Room 2": "true"
+"Dodongo Cavern Room Before Upper Lizalfos":
+  dungeon: DC
+  exits:
+    "Dodongo Cavern Upper Lizalfos": "has_explosives_or_hammer || has_blue_fire_arrows_mudwall"
```

Comment:
can you propose a change? this part of MQ DC honestly confuses me.

Reply:
`"Dodongo Cavern Upper Lizalfos": "(can_use_sticks && has(STRENGTH)) || has_explosives_or_hammer || has_blue_fire_arrows_mudwall"`

Furthermore, to get from Dodongo Cavern Room after Upper Lizalfos to Dodongo Cavern Upper Ledges this would suffice:
`"has_bombflowers || can_hammer || is_adult"`

Reply:
fixed.